### PR TITLE
Added VScode instructions

### PR DESCRIPTION
**Pull request by ilan-pinto**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](CONTRIBUTING.md)
- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/master/kogito-ide-config)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

* <b>Full Kogito testing</b> (with cloud images and operator BDD testing)  
  Please add comment: <b>Jenkins run BDD</b>  
  <b>This check should be used only if a big change is done as it takes time to run, need resources and one full BDD tests check can be done at a time ...</b>
</details>

### CODE REVIEW

File: kogito-ide-config/README.md [hunk 1]
```diff
@@ -18,4 +18,11 @@ Restart your IDE, open the _Preferences_ (or _Settings_) window again and naviga
...
 Select _Use the Eclipse Code Formatter_, then change the _Eclipse Java Formatter Config File_ to point to the `eclipse-format.xml`
 file in the `kogito-ide-config/src/main/resources/` directory.
-Make sure the _Optimize Imports_ box is ticked, and select the `eclipse.importorder` file as the import order config file.
\ No newline at end of file
+Make sure the _Optimize Imports_ box is ticked, and select the `eclipse.importorder` file as the import order config file.
+
+## VScode setup 
+
+first download the [Checkstyle for Java](https://marketplace.visualstudio.com/items?itemName=shengchen.vscode-checkstyle) plugin than right click on `kogito-ide-config/src/main/resources/eclipse-format.xml`
+and Set the *Checkstyle Configuration File*.
```
Comment (danielezonca) [inline kogito-ide-config/README.md:26]:
```suggestion
First download the [Checkstyle for Java](https://marketplace.visualstudio.com/items?itemName=shengchen.vscode-checkstyle) plugin then right click on `kogito-ide-config/src/main/resources/eclipse-format.xml` and `Set the *Checkstyle Configuration File*`. 
```

File: kogito-ide-config/README.md [hunk 2]
```diff
@@ -18,4 +18,11 @@ Restart your IDE, open the _Preferences_ (or _Settings_) window again and naviga
...
-Make sure the _Optimize Imports_ box is ticked, and select the `eclipse.importorder` file as the import order config file.
\ No newline at end of file
+Make sure the _Optimize Imports_ box is ticked, and select the `eclipse.importorder` file as the import order config file.
+
+## VScode setup 
+
+first download the [Checkstyle for Java](https://marketplace.visualstudio.com/items?itemName=shengchen.vscode-checkstyle) plugin than right click on `kogito-ide-config/src/main/resources/eclipse-format.xml`
+and Set the *Checkstyle Configuration File*. 
+
+full instruction can be found [here](https://code.visualstudio.com/docs/java/java-linting)
```
Comment (danielezonca) [inline kogito-ide-config/README.md:28]:
```suggestion
Full instruction can be found [here](https://code.visualstudio.com/docs/java/java-linting)
```

**Comment by danielezonca** [comment]
ok to test

**Comment by ilan-pinto** [comment]
@danielezonca done!

**Comment by danielezonca** [comment]
@ilan-pinto 
Can you please update your branch from master and push? We recently moved `kogito-ide-config` location (now it is inside `kogito-build` folder) and GitHub is complaining that is not possible to merge automatically

**Pull request merged**